### PR TITLE
reformat comfirmation email

### PR DESCRIPTION
**Pull request by MutegekiHenry**
### What does this PR do?
It reformats the email  for activate account. 

- Proper formatting
- use cranecloud fonts {Roboto}
- Add a confirm button 

### Need help on

- I have failed to find where the header is set to **no-reply** 

- ideas on how best to add the logo (if i store it within the api it doesnt display in the email)

- any other feedback concerning how to improve this feature is also welcome

### How to test
Run this branch locally and register a new user then check for the nature of email

### Screenshots 
Screenshot from 2020-04-22 15-17-41
Screenshot from 2020-04-22 15-16-11

### CODE REVIEW

File: templates/user/verify.html [hunk 1]
```diff
@@ -1,9 +1,141 @@
...
+                </table>
+            </td>
+        </tr>
+        <tr>
+            <td bgcolor="#f4f4f4" align="center" style="padding: 0px 10px 0px 10px;">
+                <table border="0" cellpadding="0" cellspacing="0" width="100%" style="max-width: 600px;">
+                    <tr>
+                    
+                        <td bgcolor="#ffffff" align="left" style="padding: 5px 15px 30px 15px; color: #666666; font-family: 'Roboto', Helvetica, Arial, sans-serif; font-size: 18px; font-weight: 400; line-height: 25px;">
+                            <p style="margin: 0;"> We're thrilled to have you become apart of Crane Cloud! Get ready to dive into your new account. Let's help you get started. First, you need to confirm your account. Just press the button below.</p>
```
Comment (neelxie) [inline templates/user/verify.html:102]: not **apart** guess you meant **a part**

**Comment by inno-asiimwe** [comment]
Great work @MutegekiHenry, instead of the `welcome` can we have the cranecloud logo?

File: templates/user/verify.html (hunk 1 above)
Comment (StephenAraka) [reply templates/user/verify.html:102]: Yeah, I second this. @MutegekiHenry Please take note and make that correction.

Comment (MutegekiHenry) [reply templates/user/verify.html:102]: @neelxie thanks let me change that

**Comment by MutegekiHenry** [comment]
> Great work @MutegekiHenry, instead of the `welcome` can we have the cranecloud logo?

Yes @inno-asiimwe that's what I had planned but unfortunately failed to find a way of put the logo 
since there is no link to it on the internet to access it directly, I tried adding it to the API locally and it wouldn't display in the email.  Any ideas on how to?

File: templates/user/verify.html (hunk 1 above)
Comment (MutegekiHenry) [reply templates/user/verify.html]: Done

**Comment by MutegekiHenry** [comment]
> Screenshot 2020-04-23 at 12 05 34
> you may want to fix the padding here - let it be the same for both sections

Screenshot from 2020-04-23 13-46-31
@mrwagaba the padding is uniform just that those were two different screenshots. The above can give you a better picture.

**Comment by MutegekiHenry** [comment]
> also, you're using too many exclamation marks... revise your punctuation

Let me reduce these

**Pull request merged**